R/fftrees_define.R

```diff
@@ -1,38 +1,49 @@
-#' Create definitions of FFTs.
+#' Create FFT definitions
 #'
 #' @description \code{fftrees_define} creates definitions of fast-and-frugal trees
 #' (FFTs, as an \code{FFTrees} object).
 #'
-#' @param x FFTrees.
-#' @param object FFTrees.
+#' \code{fftrees_define} usually passes passes \code{x} either
+#' to \code{\link{fftrees_grow_fan}} (to create new FFTs by applying algorithms to data) or
+#' to \code{\link{fftrees_wordstofftrees}} (if \code{my.tree} is specified).
+#' If \code{object} is provided, \code{fftrees_define} uses the trees from this \code{FFTrees} object.
+#'
+#' @param x An \code{FFTrees} object.
+#' @param object An \code{FFTrees} object.
 #'
 #' @return An \code{FFTrees} object with tree definitions.
 #'
 #' @keywords internal
 #'
+#' @seealso
+#' \code{\link{fftrees_create}} for creating \code{FFTrees} objects;
+#' \code{\link{fftrees_grow_fan}} for creating FFTs by applying algorithms to data;
+#' \code{\link{fftrees_wordstofftrees}} for creating FFTs from verbal descriptions;
+#' \code{\link{FFTrees}} for creating FFTs from and applying them to data.
+#'
 #' @import testthat
 #'
 #' @export
 
 fftrees_define <- function(x, object = NULL) {
 
-  if (is.null(object) == FALSE) {
+  if (is.null(object) == FALSE) { # an object was specified:
```

Comment:
Same as above, I don't think this comment is necessary for this line.

Reply:
Agreed & removed.